practice_projects/#Homework_pt.py

```diff
@@ -0,0 +1,34 @@
+#Homework pt.1
+
+Friend= "\nSoftware is my friend!\n"
+print (Friend)
+
+#Homework pt.2
+
+Soft = "Software is"
+my = "my friend\n"
+print (Soft,my)
+
+#Homework pt.3
+
+variable_1 = "I would like this to be a string\n"
+print (variable_1)
+
+#Homework pt.4
+
+long_string1 = "This is a super long string.\n"
+long_string2 = "        When I print this to the window, it will continue on the same line\n"
```

Comment:
Tabs in strings are denoted with "\t" similar to how a newline is "\n"
This should actually look like:
`long_string2 = "\tWhen I print this to the window, it will continue on the same line\n"`

Try it out!

Reply:
I see! Thankyou way less jank.